lib/trento/users.ex

```diff
@@ -257,10 +258,18 @@ defmodule Trento.Users do
     Map.put(attrs, :analytics_enabled_at, DateTime.utc_now())
   end
 
-  defp maybe_enable_analytics(attrs) do
+  defp maybe_enable_analytics(%{analytics_enabled: false} = attrs) do
     Map.put(attrs, :analytics_enabled_at, nil)
   end
 
+  defp maybe_enable_analytics(attrs), do: attrs
+
+  defp maybe_accept_analytics_eula(%{analytics_eula_accepted: true} = attrs) do
```

Comment:
nitpick:
maybe for our future selves, we can put a comment on top of `maybe_accept_analytics_eula` saying that we only can put this value to `true` and not false, as this is one time change (at least by now).
What do you think?